annotator/src/main/java/org/cbioportal/models/AnnotatedRecord.java

```diff
@@ -58,7 +59,8 @@
         HEADER.add("Protein_position");
         HEADER.add("Codons");        
         HEADER.add("Hotspot");
-    }       
+        HEADER.add(HEADER.indexOf("Variant_Classification"), "Consequence");
```

Comment:
We want to replace the column `Variant_Classification` with `Consequence`? I thought we just wanted to add the `Consequence` column?